ui/src/i18next.ts

```diff
@@ -25,12 +26,10 @@ const resources = {
   nl,
   it,
   fi,
+  ca,
 };
 
-function format(value: any, format: any, lng: any) {
-  if (format === 'uppercase') return value.toUpperCase();
-  return value;
-}
+const format = (value: any, format: any, lng: any) => format === 'uppercase' ? value.toUpperCase() : value;
```

Comment:
Could you revert this, I can't stand lambda functions. The conditional ternary is good tho.